Core/EventHandlers.lua

```diff
@@ -1182,19 +1178,15 @@ function R:OnEvent(event, ...)
 			end
 		end
 
-		-- Handle opening Broken Bell & Skyward Bell (Shadowlands, Bastion nodes for Soothing Vesper toy)
+		-- Handle opening Broken Bell & Skyward Bell (Shadowlands, Bastion nodes for Soothing Vesper (toy) & Gilded Wader (pet))
 		if
 			Rarity.isFishing and Rarity.isOpening and Rarity.lastNode and
 				(Rarity.lastNode == L["Broken Bell"] or Rarity.lastNode == L["Skyward Bell"])
 		 then
-			local names = {"Soothing Vesper"}
-			if (Rarity.lastNode == L["Broken Bell"]) then
-				Rarity:Debug("Detected Opening on " .. L["Broken Bell"] .. " (method = SPECIAL)")
-			elseif (Rarity.lastNode == L["Skyward Bell"]) then
-				Rarity:Debug("Detected Opening on " .. L["Skyward Bell"] .. " (method = SPECIAL)")
-			end
+			local names = {"Soothing Vesper", "Gilded Wader"}
+			Rarity:Debug("Detected Opening on " .. Rarity.lastNode .. " (method = SPECIAL)")
```

Comment:
Same as above; maybe a ``tostring(Rarity.lastNode)`` would be in order? It's verbose, but if there's no separate validation this could be an issue?